src/mmw/js/src/core/views.js

```diff
@@ -154,11 +149,36 @@ var MapView = Marionette.ItemView.extend({
 
         map.addControl(new L.Control.Zoom({position: 'topright'}));
         addLocateMeButton(map, maxZoom, maxAge);
-        map.addLayer(tileLayer);
         map.addLayer(areaOfInterestLayer);
         map.addLayer(modificationsLayer);
         map.setView([40.1, -75.7], maxZoom); // center the map
 
+         $.get('/base-layers/')
+            .done(function(response) {
+                var baseLayers = _.mapObject(response, function(layerData) {
+                    return new L.TileLayer(layerData.url, {
+                        attribution: layerData.attribution || '',
+                        maxZoom: layerData.maxZoom || 18
+                    });
+                }),
+                    defaultLayerName = _.findKey(response, function(layerData) {
+                        return layerData.default;
+                    });
+
+                L.control.layers(baseLayers).addTo(map);
```

Comment:
I don't mind the hover / collapse effect too much, although I should probably test it on a touch device. What bothers me more is how different the layer control looks compared to other icons:

![image](https://cloud.githubusercontent.com/assets/1430060/8573149/14d09264-255e-11e5-947c-5216e014cfd2.png)